packages/vue/src/components/molecules/SfNotification/SfNotification.stories.js

```diff
@@ -0,0 +1,205 @@
+        ["$notification--info-color", "$c-dark-primary", "text and icon color for info version"],
+        ["$notification--warning-color", "#d12727", "text and icon color for warning version"],
+        ["$notification--alert-color", "#0468db", "text and icon color for alert version"],
+        ["$notification--success-color", "#128830", "text and icon color for success version"],
+        ["$notification-mobile-padding", "$spacer-big", "padding on mobile"],
+        ["$notification-background-size", "cover", "size of the background"],
+        ["$notification-background-position", "bottom left", "position of the background"],
+        ["$notification__title-font-size", "3rem", "font size of the title"]
+    ]
+};
+
+// use this to document events
+const cssTableConfig = {
+    tableHeadConfig: ["NAME", "DESCRIPTION"],
+    tableBodyConfig: [
+        [".sf-notification--right", "aligns content to the right"],
+        [".sf-notification--info", "to set black color for notification message"],
+        [".sf-notification--alert", "to set blue color for notification message"],
+        [".sf-notification--warning", "to set red color for notification message"],
+        [".sf-notification--success", "to set green color for notification message"]
+    ]
+};
+
+storiesOf("Molecules|Notification", module)
+    .addDecorator(withKnobs)
+    .add(
+        "basic",
+        () => ({
+            data() {
+                return {
+                    isNotificationOpen: true
+                };
+            },
+            props: {
+                title: {
+                    default: text("title (prop)", "Added to Cart")
+                },
+                message: {
+                    default: text("message (prop)", "This is informative message for the user. View cart")
+                },
+                // level: {
+                //     default: number("level (prop)", 2)
+                // },
+                icon: {
+                    default: text("icon (prop)", "added_to_cart")
+                },
+                type: {
+                    default: select("type (prop)", [
+                        "alert",
+                        "warning",
+                        "info",
+                        "success"
+                    ])
+                }
+            },
+            components: { SfNotification, SfIcon },
+            template: `
+            <transition :name="transitionNotification">
+                <SfNotification
+                    class="sf-notification sf-notification__container" 
+                    :visible="isNotificationOpen"
+                    @close="isNotificationOpen = false"
+                    :title="title"                    
+                    :massage="massage"
+                    :icon="icon"  
+                    :type="type"            
+                >
+                    <SfIcon class="sf-notification__close" icon="cross" color="white" size="14px" @click="close" /> 
```

Comment:
```suggestion
```

Reply:
Should I remove closing icon completely?

Reply:
@AdamPawlinski you Should remove this icon from story. All elements should be implemented inside SfNotification. You have cross inside SfNotification.html :D